### PR TITLE
Add Mentor page

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #122 

Hlavně tato stránka: https://github.com/pehapkari/pehapkari.cz/pull/151/files#diff-718abef4e0770286eb6cbf30eea9d2b6

screenshot from 2017-01-02 15-57-53

### CODE REVIEW

File: source/_data/mentors.neon [hunk 1]
```diff
@@ -0,0 +1,10 @@
+mentors:
+    1:
+        name: "Milan Šulc"
+        location: "Hradec Králové"
+        # email, phone, Twitter, FB..., what you prefer the best
+        contact: "ulcmil@gmail.com"
```
Comment (f3l1x) [inline source/_data/mentors.neon]: sulcmil@gmail.com

File: source/mentori.latte [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                <td>
+                    <nobr><a n:tag-ifset="$mentor['link']" href="{$mentor['link']}">{$mentor['name']}</a></nobr>
+                </td>
+                <td>
+                    <nobr>{$mentor['location']}</nobr>
+                </td>
+                <td>
+                    <strong n:tag-if="strlen($mentor['intro_messsage']) < 140">{$mentor['intro_messsage']}</strong>
+                </td>
+                <td>{$mentor['contact']}</td>
```
Comment (f3l1x) [inline source/mentori.latte]: Maybe, email might be default contact way. And we could also add a `<a href="mailto:{$mentor['contact']}>">{$mentor['contact']}</a>`.

What about twitter / github / facebook icons? Font-awesome?

- http://fontawesome.io/icon/twitter/
- http://fontawesome.io/icon/facebook/
- http://fontawesome.io/icon/github/

:-) 

`<td n:ifset="$mentor['twitter']">{$mentor['twitter']}</td>`

Comment (TomasVotruba) [inline source/mentori.latte]: 1. Good idea with email. Less thinking what to put there and makes sense.

2. The table is quite narrow and the main message should be what do you mentor, not to list all socials sites. I'm sure everyone would put there all those 3 + website + linkein... spam spam.

Just pick the one you have the best and place it under name.

Comment (TomasVotruba) [inline source/mentori.latte]: 1. integrated

**Comment by TomasVotruba** [comment]
todo: add myself as mentor about few topics

**Comment by TomasVotruba** [comment]
@tomasfejfar Thaks for adding!

**Pull request merged**